pegasus/sites.v3/code.org/public/css/jobs.css

```diff
@@ -1,7 +1,7 @@
 .lever-job-tag {
-  margin-left: 10px;
+    margin-left: 10px;
 }
-
+  
```

Comment:
[NIT] Intentional spacing changes? You may be able to configure your editor to remove spaces that end lines and to auto-format tabs.

Reply:
Ah yes, I will fix this for the future. I won't be too worried about this particular case (since the jobs.css file will be deleted at some point in the near future, anyways)